tasks/managed/push-sbom-to-s3/push-sbom-to-s3.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-sbom-to-s3
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: |
+    Push unuploaded SBOMs to S3 to retry the Atlas push.
```

Comment:
This description should preferably match what's in the README. Sometimes it makes sense to elaborate further in the README, but the start should typically be the same.

But I see that Johnny suggests to have all of this in upload-sbom-to-atlas, so let's see about that first.